scripts/build-config-fixtures.js

```diff
@@ -0,0 +1,52 @@
+import yazl from 'yazl'
+import fs from 'node:fs/promises'
```

Comment:
Nitpick: this will work but I think we should import `fs` as a module, not its default export.

```suggestion
import * as fs from 'node:fs/promises'
```